microsoft-edge/devtools-guide-chromium/memory-problems/memory-101.md

```diff
@@ -23,222 +23,175 @@ ms.date: 07/21/2023
 
-*  Node 1 dominates node 2.
-*  Node 2 dominates nodes 3, 4 and 6.
-*  Node 3 dominates node 5.
-*  Node 5 dominates node 8.
-*  Node 6 dominates node 7.
+That is, the retained size of an object is the amount of memory that would be regained if the object and all of its dependent objects were removed from the memory graph.
 
-![Dominator tree structure](./memory-101-images/dominatorsspanning.png)
+The retained size can't be smaller than the shallow size.
 
-In the following figure, node `#3` is the dominator of node `#10`.  But node `#7` also exists in every simple path from the garbage collection root **GC** to node `#10`. Therefore, an object `B` is a dominator of an object `A` if object `B` exists in every simple path from the root to the object `A`.
+When an object is retained by multiple nodes, the object's size appears in the retained size of the retainer node that has the shortest path to the GC root.
 
-Node `GC` dominates nodes `#1`, `#3`, and `#11`:
+<!-- ------------------------------ -->
+#### Distance
 
-![Node GC dominates nodes #1, #3, and #11](./memory-101-images/dominators-00.png)
- 
-Node `#3` is dominated by node `GC` and dominates node `#7`:
+The _distance_ of an object in the JavaScript heap is the number of nodes on the shortest path between the object and the GC root. The shorter the distance, the more likely it is that this object plays an important role in the memory usage of the webpage.
 
-![Node #3 is dominated by node GC and dominates node #7](./memory-101-images/dominators-01.png)
- 
-Node `#7` is dominated by node `#3` and dominates nodes `#8`, `#9`, and `#10`:
 
-![Node #7 is dominated by node #3 and dominates nodes #8, #9, and #10](./memory-101-images/dominators-02.png)
- 
-Node `#8` is dominated by node `#7` and doesn't dominate any nodes:
+<!-- ====================================================================== -->
+## Retainers
 
-![Node #8 is dominated by node #7 and doesn't dominate any nodes](./memory-101-images/dominators-03.png)
- 
-Node `#10` is dominated by node `#7` and doesn't dominate any nodes:
+An object's retainers are the others objects that hold references to it. The **Retainers** section of the **Memory** tool shows the objects that hold references to the object selected in the **Summary** view.
 
-![Node #10 is dominated by node #7 and doesn't dominate any nodes](./memory-101-images/dominators-04.png)
- 
-Node `#11` is dominated by node `#1` and doesn't dominate any nodes:
+The **Retainers** section of the **Memory** tool is sorted by distance by default, which means that the simplest retaining paths for an object are shown first.
 
-![Node #11 is dominated by node #1 and doesn't dominate any nodes](./memory-101-images/dominators-05.png)
+Any object with no retainers can be discarded by the browser’s garbage collector, which reduces memory usage.
```

Comment:
```suggestion
Any object with no retainers can be discarded by the browser's garbage collector, which reduces memory usage.
```
straight quotes per Contrib Guide, though skeptical that it makes a difference

GLOBAL nice to do (a separate Comment): change any curly quotes in article to straight quotes.